vllm/v1/worker/gpu_model_runner.py

```diff
@@ -3466,9 +3561,6 @@ def get_kv_cache_spec(self) -> dict[str, KVCacheSpec]:
                 self.shared_kv_cache_layers[layer_name] = kv_tgt_layer
                 continue
 
-            # TODO: Support other attention modules, e.g., cross-attention
-            # TODO(lucas): move the attention specs into the model layers like
```

Comment:
Can you keep the second TODO left by lucas?

Reply:
Yes! Sorry about that. I probably removed it by accident while resolving conflicts at some point.